src/index.js

```diff
@@ -68,6 +68,32 @@ export class Mobilitybox {
       .then(response => new MobilityboxTrip(response.data, this))
   }
 
+  /**
+   * Get a trip by its characteristics
+   * @param {object} origins_from_station MobilityboxStop or Identifier of the first station of the trip. Usually the start with: `vesputi:station:`
+   * @param {object} destination_station MobilityboxStop or Identifier of the last station of the trip. Usually the start with: `vesputi:station:`
+   * @param {object} origins_from_departure_time The departure time on the starting station as unix timestamp in milliseconds or as MobilityboxEventTime
+   * @param {object} destination_arrival_time The arrival time on the ending station as unix timestamp in milliseconds or as MobilityboxEventTime
+   * @param {String} line_name The short name of the corresponding line
+   * @return {MobilityboxTrip} found trip
```

Comment:
The return value is a promise, right? Not the MobilityboxTrip directly. That's confusing then for the user of this API.